src/index.html

```diff
@@ -1,13 +1,37 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image"></div>
+      <p class="product-card__info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <p class="product-card__code">Код товара: 195434</p>
+      <div class="product-card__rating">
+        <div class="product-card__stars">
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+          <div class="product-card__star"></div>
+        </div>
+        <div class="product-card__feedback">Отзывов: 5</div>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__text-of-price">Цена:</p>
+        <div class="product-card__total-price">69 999 грн</div>
+      </div>
+      <div class="product-card__buy">
+        <a href="/" class="product-card__link" data-qa="hover">Купить</a>
+      </div>
```

Comment:
```suggestion
      <a href="/" class="product-card__link" data-qa="hover">Купить</a>
```
Можна margin-top задати для самої лінки і не створювати додаткової обгортки